project_scrum/models/project_sprint.py

```diff
@@ -0,0 +1,135 @@
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+from odoo.tools import LazyTranslate
+
+_lt = LazyTranslate(__name__, default_lang="en_US")
+
+DATE_OPTIONS = [
+    ("1_weeks", str(_lt("1 Week"))),
+    ("2_weeks", str(_lt("2 Weeks"))),
+    ("1_months", str(_lt("1 Month"))),
+    ("2_months", str(_lt("2 Month"))),
+    ("1_years", str(_lt("1 Year"))),
+    ("2_years", str(_lt("2 Years"))),
+    ("custom", str(_lt("Custom"))),
+]
+
+
+class ProjectSprint(models.Model):
+    _name = "project.sprint"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+    _description = "Project Sprint"
+    _sql_constraints = [
+        (
+            "date_check",
+            "CHECK (date_start <= date_end)",
+            "Error: End date must be greater than start date!",
+        ),
+    ]
+
+    name = fields.Char(required=True, tracking=True)
+    user_ids = fields.Many2many(
+        comodel_name="res.users",
+        string="Members",
+        required=True,
+        domain="[('share', '=', False), ('active', '=', True)]",
+        tracking=True,
+        relation="project_sprint_user_rel",
+    )
+    description = fields.Text(tracking=True)
+    project_id = fields.Many2one(
+        comodel_name="project.project",
+        string="Project",
+        tracking=True,
+    )
+    task_ids = fields.One2many(
+        comodel_name="project.task",
+        inverse_name="sprint_id",
+        string="Tasks",
+        domain="[('project_id', '=', project_id)]",
+    )
+    date_start = fields.Date(
+        string="Start Date", default=fields.Date.today, required=True
```

Comment:
Please use `context_today` to prevent issues in countries where the time zone is negative.
```suggestion
        string="Start Date", default=fields.Date.context_today, required=True
```

Reply:
Done!